doc/concepts/morphology.rst

```diff
@@ -482,6 +482,96 @@ because it has two children: the dendrites attached to its distal end.
+   * Check that there are no duplicate ids.
+   * Check that the parent id of a sample is less than the id of the sample.
+   * Check that the parent id of a sample refers to an existing sample.
+
+In addition, all interpretations agree that a *segment* is (in the common case) constructed between a sample and
+its parent sample; and if more than 1 sample have the same parent sample, the parent sample is interpreted as a fork
+point in the morphology, and acts as the proximal point to a new branch for each of its "child" samples.
+There a couple of exceptions to these rules which are listed below.
+
+Arbor interpretation:
+"""""""""""""""""""""
+In addition to the previously listed checks, the arbor interpretation explicitly disallows SWC files where the soma is
+described by a single sample. It constructs the soma from 2 or more samples that form 1 or more segments in the segment
+tree. A *segment* is constructed between a sample and its parent, and inherits the sample's tag, regardless of
+whether or not its tag matches the parent's tag. This means that there are no gaps in the resulting segment tree.
+
+Arbor has no magic rules or transformations for the soma. It can be a single branch or multiple branches; segments
+of a different tag can connect to its distal end, proximal end or anywhere in the middle. For example, to create a
+morphology with a single segment soma; a single segment axon connected to one end of the soma; and a single segment
+dendrite connected to the other end of the soma, the following swc file can be used:
+
+.. code:: Python
+
+   # id, tag, x, y,   z, r, parent
+      1,   1, 0, 0,   0, 5, -1
+      2,   1, 0, 0,  10, 5,  1
+      3,   2, 0, 0, -10, 2,  1
+      4,   3, 0, 0, 100, 1,  2
+
+Samples 1 and 2 will form the soma; samples 1 and 3 will form the axon, connected to the soma at the proximal end;
+samples 2 and 4 will form the dendrite, connected to the soma at the distal end.
+
+Allen interpretation:
+"""""""""""""""""""""
+In addition to the previously mentioned checks, the Allen interpretation expects a single-sample soma to be the first
+sample of the file and to be interpreted as a spherical soma. Arbor represents the spherical soma as a cylinder with
+length and diameter equal to the diameter of the sample representing the sphere.
+
+This interpretation also expects that samples have the same tag as their parent samples, with the exception of samples
+that have the soma sample as a parent. In this case, when a sample's parent is the soma, no *segment* is created
+between the 2 samples; instead there is a gap in the morphology (represented electrically as a zero-resistance wire).
+Samples with the soma as a parent start new segments, that connect to the distal end of the soma if they are dendrites,
+or to the proximal end of the soma if they are axons or apical dendrites. Only axons, dendrites and apical dendrites
+(tags 2, 3 and 4 respectively) are allowed in this interpretation.
```

Comment:
Om the ris of being annoying, perhaps add the soma?

Reply:
Yes, I'll do that